src/lib/core/fiber.h

```diff
@@ -459,24 +460,13 @@ cord_slab_cache(void);
  * internal structures may be changed in a future, but
  * <box_region_*>() functions will remain API and ABI compatible.
  *
- * Data allocated on the region are guaranteed to be valid until
- * a call of a function from the certain set:
- *
- * - Related to transactions.
- * - Ones that may cause box initialization (box.cfg()).
- * - Ones that may involve SQL execution.
- *
- * FIXME: Provide more strict list of functions, which may
- * invalidate the data: ones that may lead to calling of
- * fiber_gc().
- *
- * It is safe to call simple box APIs around tuples, key_defs and
- * so on -- they don't invalidate the allocated data.
- *
  * Each fiber has its own box region. It means that a call of,
  * say, <box_region_used>() will give its own value in different
  * fibers. It also means that a yield does not invalidate data in
  * the box region.
+ *
+ * Before version 2.10 one can only use a subset of API between
+ * box_region_used and box_region_truncate.
```

Comment:
I don't understand this comment. What "subset of API"?

Anyway, I'm not sure we should backport this to 2.10.

Reply:
Rewrote the comment as discussed f2f.